packages/account/src/Components/trading-assessment/trading-assessment-radio-buttons.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Field } from 'formik';
+import { Text, RadioGroup } from '@deriv/components';
+
+const TradingAssessmentRadioOption = ({ text, list, onChange, values, form_control, setEnableNextSection }) => {
```

Comment:
The file is named `trading-assessment-radio-buttons` but the component has a different name?